packages/build/tests/plugins/fixtures/functions_src_dynamic/plugin.js

```diff
@@ -6,16 +6,19 @@ const { promisify } = require('util')
 const pMkdir = promisify(mkdir)
 const pRmdir = promisify(rmdir)
 
-const DEFAULT_FUNCTIONS_SRC = 'netlify-automatic-functions'
+const DEFAULT_FUNCTIONS_SRC_PARENT = 'netlify'
```

Comment:
😖 This is super icky, but Node 8 does not support the `recursive` flag in `fs.mkdir` and this was the least terrible way of keeping the fixtures simple.

Reply:
https://github.com/netlify/build/pull/2188/commits/0c99646786faa0b1ad497e228774107f3c8bd33e is perhaps slightly less icky?

Reply:
Looks good!
Another reason to drop [Node 8](https://github.com/netlify/team-dev/issues/2).